lib/SIL/SILFunctionType.cpp

```diff
@@ -2213,12 +2225,26 @@ namespace {
         substParams.push_back(subst(origParam));
       }
 
-      return SILFunctionType::get(nullptr,
-                                  origType->getExtInfo(),
-                                  origType->getCalleeConvention(),
-                                  substParams, substResults,
-                                  substErrorResult,
-                                  getASTContext());
+      Optional<ProtocolConformanceRef> witnessMethodConformance;
+      if (auto conformance = origType->getWitnessMethodConformanceOrNone()) {
+        if (conformance->isAbstract()) {
```

Comment:
@slavapestov I don't understand what substitution I'm meant to being here to convert the abstract conformance into a concrete one (and even for the concrete case, I'm just guessing).